webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartling.java

```diff
@@ -648,6 +656,22 @@ private void processPullBatch(
                                 new SmartlingClientException(
                                     "Error with download from Smartling, file content string is not present."));
 
+                if (options.isDeltaPull()
+                    && isFileEqualToPreviousRun(
+                        thirdPartyFileChecksumRepository,
+                        repository,
+                        locale.getLocale(),
+                        fileName,
+                        fileContent)) {
+                  logger.info(
```

Comment:
Could we add a metric here to track short-circuited vs. non short-circuited pulls (when isDeltaPull() is enabled), with tags for repoName, locale & file name - so that we can see the percentage difference that this makes in practice?

shortCircuited=True/False can be a tag on that one metric.

Reply:
I can add a true/false tag to the existing timer metric that is wrapping the download file from Smartling and import the text units.

New tag will allow us to differentiate the difference between pulls with delta pull enabled vs those with it disabled.

Reply:
If it's an existing metric, we need at least two new tags: deltaPullEnabled and fileShortCircuited, both booleans - to be able to accurately tell where it had an impact.

Does the existing metric report the other tags already? (repoName, locale & file name)

Reply:
Existing processPullBatch metric already has the repository and the locale as tags, we don't tag on the filename. Current implementation tracks the timing in a lamda so we don't have the short-circuited outcome available. 

It might be easier to just record this as a new counter stat with tags for delta pull enabled and shortCircuited